sw/device/lib/testing/test_framework/ottf_test_config.h

```diff
@@ -32,6 +32,14 @@ typedef struct ottf_test_config {
    */
   bool enable_concurrency;
 
+  /**
+   * Indicates that this test will utilize the UART to receive commands from
+   * a test harness and that the UART should enable software flow control.
+   * Note that requesting flow control will unmask the external interrupt and
+   * enable interrupt handling before `test_main` begins.
+   */
+  bool want_flow_control;
```

Comment:
nit: maybe `enable_uart_flow_control` would be more descriptive / consistent? in case we ever add flow control switches for other comm interfaces? (fine as is tho too).

Nice to have this toggle switch!

Reply:
Done.